doc/source/tune/examples/tune_analyze_results.ipynb

```diff
@@ -0,0 +1,912 @@
+       "3                  52       0.978125\n",
+       "4                  62       0.987500\n",
+       "5                  57       0.971875\n",
+       "6                  38       0.978125\n",
+       "7                  77       0.981250\n",
+       "8                  92       0.931250"
+      ]
+     },
+     "execution_count": 7,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "worst_result_df = result_grid.get_dataframe(\n",
+    "    filter_metric=\"mean_accuracy\", filter_mode=\"max\"\n",
+    ")\n",
+    "worst_result_df[[\"training_iteration\", \"mean_accuracy\"]]"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "5eeffb9a-9f50-4036-8925-82e45d304166",
+   "metadata": {},
+   "source": [
+    "## Trial-level Analysis: Working with an individual `Result`"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "ae8b394d-fedc-4522-990a-2cf50fbf4276",
+   "metadata": {},
+   "source": [
+    "Let's take a look at the result that ended with the best `mean_accuracy` metric. By default, `get_best_result` will use the same metric and mode as defined in the `TuneConfig` above. However, it's also possible to specify a new metric/order in which results should be ranked."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 8,
+   "id": "d7e0ce5b-e278-40d1-aee6-e2840a9fa77f",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from ray.air import Result\n",
+    "\n",
+    "# Get the result with the maximum test set `mean_accuracy`\n",
+    "best_result: Result = result_grid.get_best_result()\n",
+    "\n",
+    "# Get the result with the minimum `mean_accuracy`\n",
+    "worst_performing_result: Result = result_grid.get_best_result(\n",
+    "    metric=\"mean_accuracy\", mode=\"min\"\n",
+    ")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "58045cd9-3894-48b8-8196-1c81a00feb7d",
+   "metadata": {},
+   "source": [
+    "We can examine a few of the properties of the best `Result`. See the [API reference](result-docstring) for a list of all accessible properties."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 9,
+   "id": "1f1c3d9b-ec32-439d-9e8e-e9afb58f1331",
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "{'lr': 0.0876741358605454, 'momentum': 0.8, 'should_checkpoint': True}"
+      ]
+     },
+     "execution_count": 9,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "best_result.config"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "8fac8cf8-67b6-4f84-9227-19db05957d5c",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "best_result.log_dir"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "88712bb6-2015-4417-966a-119caa31fb2a",
+   "metadata": {},
+   "source": [
+    "The `log_dir` will look something like: `PosixPath('~/ray_results/tune_analyzing_results/train_mnist_...')`\n",
```

Comment:
instead of mentioning this, the output of `best_result.log_dir` cell will should actually show this right?